.circleci/config.yml

```diff
@@ -229,9 +244,28 @@ jobs:
       - store_test_results:
          path: /root/project/e2e
 
+
+  delete-helm-preview:
+    <<: *build_config
+
+    environment:
+      GHA_PREVIEW_NR: << pipeline.parameters.GHA_Meta >>
```

Comment:
and the call this PREVIEW_NR

Reply:
but its the number passed by github actions , if a different context the PREVIEW_NR might be something different , i think its good to be explicit in this case , and alert the user to the fact that the number is the data being passed via github